### PR TITLE
Switch from METADATA to registrator

### PR DESCRIPTION
**Pull request by kescobo**
With the [new package registration system](https://discourse.julialang.org/t/switching-package-registration-systems-soon/22677/144), REQUIRE is out and Project.toml is in.

This is an annoying but necessary task, so I thought I'd just do it. I used [this script](https://github.com/JuliaLang/Pkg.jl/pull/1133), and added a version (`0.14.0`, since I figured this warranted a major version bump).

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+
+[compat]
+Distances = "≥ 0.8.0"
+NearestNeighbors = "≥ 0.0.3"
+StatsBase = "≥ 0.9.0"
+julia = "≥ 1.0.0"
```
Comment (nalimilan) [inline Project.toml]:
```suggestion
julia = "1"
```

**Comment by nalimilan** [comment]
> `0.14.0`, since I figured this warranted a major version bump

Does it? I don't think so, as nothing changes at all for users (the registries already generated the Project.tool file automatically under the hood).

**Comment by alyst** [comment]
Before the new release, I would like to merge #168, which is a (small) API change warranting a major version bump.
BTW, @nalimilan could you please take a look at #168?

**Comment by kescobo** [comment]
>Does it? I don't think so, as nothing changes at all for users

Fair enough - I definitely don't feel strongly about it. Should I change to 0.13.2 then?

**Comment by kescobo** [comment]
Fine by me either way. Let me know if you want me to change it or leave it alone :-)

**Comment by alyst** [comment]
I think the consensus now is that the next release we will 0.13.2.
@kescobo Could you please commit the suggested change, so that I can merge the PR?

**Comment by kescobo** [comment]
@alyst All set!

**Comment by alyst** [comment]
@kescobo Thanks! Actually, I've never used Registrator before :) Should I add the comment `@JuliaRegistrator register + release notes` and then merge?

**Comment by kescobo** [comment]
@alyst Evidently not? 😆 

I've only used BioJulia's registrator, but I did it in an issue after the PR was merged. Seems like it makes the most sense to do in in the PR once it's merged but ¯\_(ツ)_/¯

**Comment by alyst** [comment]
Also I should be careful calling the registrator by name, since it doesn't understand the quotes. :)
I'll try commenting the head commit after merging the PR.

**Pull request merged**